### PR TITLE
OMS-0013-0014-0015-0016-Code-Commit

### PR DESCRIPTION
**Pull request by pvrgithub**
Yet to write mock and acceptance. Please don't merge this now.

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+info:
+    description: "Inventory Management application"
+spring:
+  application:
+    name: inventory-service
+  data:
+    mongodb:
+        uri: mongodb://localhost:27017/OMS
```
Comment (ashokpivotal) [inline src/main/resources/application.yml]: port , uri: mongodb://localhost:27017/OMS  needs to be removed .

File: src/acceptance/groovy/InventoryCRUDAccSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import spock.lang.Shared
+import spock.lang.Specification
+
+class InventoryCRUDAccSpec extends Specification {
+
+    @Shared
+    RESTClient restClient
+
+    @Shared
+    String localhost = "http://localhost:9099/inventory"
```
Comment (ashokpivotal) [inline src/acceptance/groovy/InventoryCRUDAccSpec.groovy]: common config file required

File: src/main/resources/application.yml (hunk 1 above)
Comment (chnmerrill) [reply src/main/resources/application.yml]: OK :). it will be removed. I have raised the PR so that I can pull this from home

**Comment by pvrgithub** [comment]
Acceptance and mock tests are done for inventory module

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  common:
+    web:
+      config:
+        swagger:
+          controllerPackage: com.oms.inventory.controller
+          author: PVR
+          apiName: Inventory-Service
+          apiVersion: 1.0.0
+          apiDescription: API for Inventory-service
+logging:
```
Comment (ashokpivotal) [inline src/main/resources/application.yml:26]: check this logging config

File: src/main/java/com/oms/inventory/controller/InventoryController.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import com.oms.inventory.model.InventoryProduct;
+import com.oms.inventory.model.InventoryRequest;
+import com.oms.inventory.model.InventoryResponse;
+import com.oms.inventory.model.InventoryUpdate;
+import com.oms.inventory.service.InventoryService;
+import com.oms.inventory.service.InventoryServiceImpl;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
```
Comment (ashokpivotal) [inline src/main/java/com/oms/inventory/controller/InventoryController.java:12]: add Header comments

**Review by ashokpivotal — APPROVED** [review]

**Pull request merged**